### ISSUE TITLE
"My Profile" modal controls must be accessible by keyboard

### ISSUE THREAD

**Issue by will0684** [gcdevops/HRWhiteListing#193]
The following highlighted controls are not accessible by keyboard:

<img width="978" alt="Screen Shot 2020-04-15 at 8 43 06 AM" src="https://user-images.githubusercontent.com/31868510/79339239-92501d80-7ef6-11ea-9989-866fcc075f00.png">

You can read about this WCAG2.0 success criterion here:

https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html
